material-ui-superselectfield/material-ui-superselectfield.d.ts

```diff
@@ -30,7 +37,7 @@ declare module "material-ui-superselectfield" {
         menuStyle?: Object;
         menuGroupStyle?: Object;
         menuFooterStyle?: Object;
-        menuCloseButton?: any;
+        footerActionsRenderer?: any;
```

Comment:
I'm not seeing this in the README, something separate?

Reply:
From the release notes I saw

`Replaced menuCloseButton with footerActionsRenderer`

Reply:
Breaking change under release 1.6.0

https://github.com/Sharlaan/material-ui-superselectfield/releases

Reply:
👍